2] loto/main.py

```diff
@@ -0,0 +1,45 @@
+import random
+
+
+def get_user_numbers():
+    user_numbers = set()
+    while len(user_numbers) < 6:
+        try:
+            user_input = input("Enter a number (1-49): ")
+            number = int(user_input)
+            if number < 1 or number > 49:
+                print("Number must be between 1 and 49.")
+            elif number in user_numbers:
+                print("You have already entered this number.")
+            else:
+                user_numbers.add(number)
+        except ValueError:
+            print("That's not a valid number.")
+    return sorted(user_numbers)
+
+
+def draw_lotto_numbers():
+    return sorted(random.sample(range(1, 50), 6))
+
+
+def compare_numbers(user_numbers, lotto_numbers):
+    return len(set(user_numbers).intersection(set(lotto_numbers)))
```

Comment:
tohle mi prijde nevhodne predelavat to tady na set abys mohl udelat intersetcion. Mnohem lepsi by bylo kdyzby ty funkce co ty data pripravuji uz set vratily